spotify/job/test_dates.py

```diff
@@ -0,0 +1,24 @@
+import datetime as dt
+import unittest
+
+from job.dates import DateRange
+
+class TestDateRange(unittest.TestCase):
+    def setUp(self):
+        self.start = dt.datetime(2023, 1, 1)
+        self.end = dt.datetime(2023, 1, 5)
+        self.daterange = DateRange(self.start, self.end)
+
+    def test_iterator(self):
+        expected_output = [(dt.datetime(2023, 1, 1), dt.datetime(2023, 1, 5)),
```

Comment:
why is the output a range? shouldn't it be just one day or the range of start===end?